src/modules/Aieracast/index.tsx

```diff
@@ -49,6 +134,20 @@ export function AieracastUI(props: AieracastUIProps): ReactElement {
 
     const updateGlobalSearch = debounce((v: string) => setGlobalSearchState(v), 250);
 
+    const onDragEnd = useCallback(
+        (dragEvent: DragEndEvent) => {
+            const { active, over } = dragEvent;
+            if (active.id !== over?.id && over) {
```

Comment:
Super pedantic and practically moot, but if `over` is first, you'd avoid an unnecessary equality check: `(over && active.id !== over?.id)`